### ISSUE TITLE
Change theme has some glitches

### ISSUE THREAD

**Issue by jibedoubleve** [jibedoubleve/lanceur-bis#94]
![image](https://user-images.githubusercontent.com/2678545/213121263-91828332-77fc-4804-abc7-ab012333d5c1.png)

**Comment by jibedoubleve** [comment]
# Invalid

This is not a problemof Lanceur but a problem of the Alias

User set up alias like this
| Parameter | Value|
|---|---|
| File Name|powershell
|Arguments| -NoProfile Set-ItemProperty -Path HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Themes\Personalize -Name SystemUsesLightTheme -Value 0|

The problem is that it is only changing the value of the application theme. To make the theme change bettern the script should change application theme and Windows theme

In other words, the script should be
```powershell
Set-ItemProperty -Path HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Themes\Personalize -Name SystemUsesLightTheme -Value 0

Set-ItemProperty -Path HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Themes\Personalize -Name AppsUseLightTheme -Value 0
```

# Todo

- [ ] Decide whether create a `macro` or a `plugin`
- [ ] Create a xxx for the light theme
- [ ] Create a  xxx for the dark theme

**Comment by jibedoubleve** [comment]
Maybe this can be interesting info:

https://stackoverflow.com/questions/17431883/set-datatemplate-in-template-selector-as-dynamic-resource
